src/main/resources/application-dev.yml

```diff
@@ -1,4 +1,4 @@
 offender:
   search:
     endpoint:
-      url: https://probation-offender-search-dev.hmpps.service.justice.gov.uk
+      url: ${probation.offender.search.endpoint.url}
```

Comment:
You can set this specifically as `https://probation-offender-search-dev.hmpps.service.justice.gov.uk` if you want - then you won't need to set environment variables for local dev, it's only `application.yml` that needs to be environment driven. :)

Reply:
Lol! Of course ... insert face palm here!

Reply:
Done